code/tracing/categories.cpp

```diff
@@ -13,6 +13,7 @@ bool Category::usesGPUCounter() const {
 }
 
 Category LuaOnFrame("LUA On Frame", true);
+Category LuaHooks("LUA hooks", false);

```

Comment:
Since Lua can trigger graphics intesive stuff you might want to turn this into a GPU trace as well.

Reply:
Honestly I don't really know how our graphics tracing works and if this would be a reasonable thing to do, so I'd rather leave it like this unless there's a clear benefit.

Reply:
I think it would be reasonable to do that. The additional overhead is relatively minor and it might be useful at some point so it would be better to have it than not have it.

Reply:
OK, I switched it on but on your head be it!!